botocore/utils.py

```diff
@@ -35,6 +35,11 @@
 # These are chars that do not need to be urlencoded.
 # Based on rfc2986, section 2.3
 SAFE_CHARS = '-._~'
+LABEL_RE = re.compile('[a-z0-9][a-z0-9\-]*[a-z0-9]')
```

Comment:
I just copied and pasted this from handlers.py . I would be inclined not to change this to avoid accidental changes, if there is no bugs with it already.